mediapipe/examples/desktop/face_mesh_dll/face_mesh_cpu.cpp

```diff
@@ -36,26 +51,26 @@ int main(int argc, char **argv) {
       LOG(INFO) << "Ignore empty frames from camera.";
       continue;
     }
+
     cv::Mat camera_frame;
     cv::cvtColor(camera_frame_raw, camera_frame, cv::COLOR_BGR2RGB);
     cv::flip(camera_frame, camera_frame, /*flipcode=HORIZONTAL*/ 1);
 
-    std::unique_ptr<std::vector<std::vector<cv::Point2f>>> multi_face_landmarks(
-        reinterpret_cast<std::vector<std::vector<cv::Point2f>> *>(
-            FaceMeshDetector_ProcessFrame2D(faceMeshDetector, camera_frame)));
+    int faceCount =
```

Comment:
nit: what is the need for the new line here?

Reply:
Just strange auto-formatting)